po4a.cfg

```
@@ -1,10 +1,10 @@
-[po4a_langs] pl
+[po4a_langs] es
 [po4a_paths] translations/$master/$master.pot $lang:translations/$master/$lang.po
 
 [options] --addendum-charset=UTF-8 \
           --localized-charset=UTF-8 \
           --master-charset=UTF-8 \
-          --master-language=en_US \
+          --master-language=es_ES \
```

```suggestion
          --master-language=en_US \
```
This is the language you're translating from, not to.

---

ok, thank you for info

---

May I ask, what is it you're trying to achieve here? We already have automation that generates all the po files.

---

Hi, sorry, I'm a newbie to this. I didn't know that po were generated automatically. Anyway, I have created a process that creates the es.po files and automatically translates them using Google Translator (with some problems at the moment). Now it would be necessary to review them and correct the translation errors or complete what has not been translated. A greeting.